pkg/api/types.go

```diff
@@ -173,22 +173,22 @@ type MasterProfile struct {
 
 // AgentPoolProfile represents an agent pool definition
 type AgentPoolProfile struct {
-	Name                string `json:"name"`
-	Count               int    `json:"count"`
-	VMSize              string `json:"vmSize"`
-	OSDiskSizeGB        int    `json:"osDiskSizeGB,omitempty"`
-	DNSPrefix           string `json:"dnsPrefix,omitempty"`
-	OSType              OSType `json:"osType,omitempty"`
-	Ports               []int  `json:"ports,omitempty"`
-	AvailabilityProfile string `json:"availabilityProfile"`
-	StorageProfile      string `json:"storageProfile,omitempty"`
-	DiskSizesGB         []int  `json:"diskSizesGB,omitempty"`
-	VnetSubnetID        string `json:"vnetSubnetID,omitempty"`
-	Subnet              string `json:"subnet"`
-	IPAddressCount      int    `json:"ipAddressCount,omitempty"`
-
-	FQDN             string            `json:"fqdn,omitempty"`
-	CustomNodeLabels map[string]string `json:"customNodeLabels,omitempty"`
+	Name                string            `json:"name"`
+	Count               int               `json:"count"`
+	VMSize              string            `json:"vmSize"`
+	OSDiskSizeGB        int               `json:"osDiskSizeGB,omitempty"`
+	DNSPrefix           string            `json:"dnsPrefix,omitempty"`
+	OSType              OSType            `json:"osType,omitempty"`
+	Ports               []int             `json:"ports,omitempty"`
+	AvailabilityProfile string            `json:"availabilityProfile"`
+	StorageProfile      string            `json:"storageProfile,omitempty"`
+	DiskSizesGB         []int             `json:"diskSizesGB,omitempty"`
+	VnetSubnetID        string            `json:"vnetSubnetID,omitempty"`
+	Subnet              string            `json:"subnet"`
+	IPAddressCount      int               `json:"ipAddressCount,omitempty"`
+	Distro              string            `json:"distro,omitempty"`
```

Comment:
@amanohar  - How and where OSType defined? 

I am modifying cosnt.go with below but it doesn't work.

// the Distros supported by vlabs
const (
	CoreOS Distro = "CoreOS"
	Ubuntu  Distro  = "Ubuntu"
)